include/deal.II/differentiation/sd/symengine_scalar_operations.h

```diff
@@ -170,7 +170,7 @@ namespace Differentiation
      * valid symbols or symbolic expressions.
      *
      * @tparam ignore_invalid_symbols See the
-     * @ref add_to_symbol_map(types::substitution_map &, const Expression &)
+     * `add_to_symbol_map(types::substitution_map &, const Expression &)`
```

Comment:
Do these still link back to the referenced function?

Reply:
Yes, I still see the links.

Reply:
Awesome. Thanks a lot for the confirmation!